src/getCoinCombination.js

```diff
@@ -10,6 +10,10 @@ function getCoinCombination(cents) {
   const values = [1, 5, 10, 25];
   const coins = [0, 0, 0, 0];
 
+  if (cents < 0) {
```

Comment:
The function currently returns an array of zeros if the input 'cents' is negative. Consider whether this is the intended behavior or if an error should be thrown or a different value returned.